lib/Net/EmptyPort.pm

```diff
@@ -2,11 +2,22 @@ package Net::EmptyPort;
 use strict;
 use warnings;
 use base qw/Exporter/;
-use IO::Socket::INET;
+use IO::Socket;
 use Time::HiRes ();
 
 our @EXPORT = qw/ empty_port check_port wait_port /;
 
+our $io_socket_module_name;
```

Comment:
`my` is better.

--
tokuhirom

On Sat, Jan 3, 2015 at 12:41 PM, Tatsuhiko Miyagawa <
notifications@github.com> wrote:

> In lib/Net/EmptyPort.pm
> <https://github.com/tokuhirom/Test-TCP/pull/17#discussion-diff-22429255>:
>
> >  use Time::HiRes ();
> >
> >  our @EXPORT = qw/ empty_port check_port wait_port /;
> >
> > +our $io_socket_module_name;
>
> probably should be my then?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tokuhirom/Test-TCP/pull/17/files#r22429255>.
>